tokens/test/contracts/access/MinterAccessControlTestV2.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.8.0;
+
+import "../../../contracts/access/MinterAccessControl.sol";
+
+contract MinterAccessControlTestV2 is MinterAccessControl {
+
+    function initialize() external initializer {
+        __Ownable_init_unchained();
+        __MinterAccessControl_init_unchained();
+    }
+
+    function version() public view returns (uint256) {
```

Comment:
For version identification we use smth like that:
bytes4 constant public V2 = bytes4(keccak256("V2"));
You can see it: exchange-v2/contracts/LibOrderDataV2.sol
So may You use this constant in test
I think there is no error here, but probably we should use the same approach

Reply:
updated: https://github.com/rarible/protocol-contracts/pull/105/commits/af18208855fd179838ea47e014d696363c0df14e

Reply:
Thank You for your work, I`ll review

Reply:
@SKonstantinS waiting for your review ;)